src/target/riscv/batch.h

```diff
@@ -14,6 +14,17 @@ enum riscv_scan_type {
 	RISCV_SCAN_TYPE_WRITE,
 };
 
+/* These types are used to specify how many JTAG RTI cycles to add after a
+ * scan.
+ */
+enum riscv_scan_delay_type {
+	RISCV_SCAN_DELAY_DM,
```

Comment:
Optional - if you like:

```suggestion
	RISCV_SCAN_DELAY_BASE,
```

Reply:
Addressed.